core/templates/components/molecules/ctas/ctas.twig

```diff
@@ -0,0 +1,5 @@
+<a {{ attributes }} href="{{ link|default("https://stanford.edu") }}" class="{{ attributes.addClass('decanter-cta') }} {{ modifier_class }} {{ grid }}">
```

Comment:
You shouldn't have to add the class to attributes like this. Instead, just have the class text. eg:
`<a{{ attributes }} href="{{ link|default("https://stanford.edu") }}" class="decanter-cta {{ modifier_class }} {{ grid }}">`

Reply:
no space after the `<a` and `{{ attributes }}`

Reply:
Changed - thanks!